### ISSUE TITLE
[Bug]: sp-table-head-cell with sortable cannot be activated by keyboard

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#4472]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-table-head-cell

### Expected behavior

Should be able to be sorted using the keyboard

### Actual behavior

neither space nor enter will sort the column

### What browsers are you seeing the problem in?

Firefox, Chrome, Safari, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/table-virtualized--virtualized
2. Tab until you get to the sortable column title
3. Press enter 
4. Nothing happens
5. Press space
6. page scrolls
